blackbox/docs/release_notes/2.0.0.txt

```diff
@@ -30,8 +30,12 @@ Breaking Changes
 
    If you disable this setting, CrateDB will run with the standard feature set.
 
- - The default value of the ``node.max_local_storage_nodes`` node setting has
-   been changed to ``1`` to prevent running multiple nodes on the same data
+ - All ``node.*`` style attributes must now be written as ``node.attr.*``, with
```

Comment:
Maybe rephrase this a bit.

All *custom* ``node.*`` style attributes must now be written as [...] to
distinguish them from core node attributes like ``node.name``.

The point is that anything in `node.attr` is *not* validated, but rather it's a
custom bag of attributes one can attach to a node. The keys/values mostly don't
matter.

(Except for how it can be used to control routing allocation a bit - see our
multi-zone setup documentation).

All `node.XY` settings are validated on startup and the node won't start up
if the setting is invalid or there is a typo in the name. E.g.
    `node.sql.read_only=foobar` will fail because it expects a boolean value.
    Similar `node.nme` will cause a failure because the setting doesn't exist.